charts/tidb-operator/values.yaml

```diff
@@ -63,8 +80,6 @@ scheduler:
   logLevel: 2
   replicas: 1
   schedulerName: tidb-scheduler
-  # features:
-  # - StableScheduling=true
```

Comment:
this feature is enabled by default and I guess no one will disable it in production, so I think we don't need to support `scheduler.features` for backward compatibility

Reply:
Breaking backward compatibility should be noted in the PR description.